cmd/podman/import.go

```diff
@@ -69,8 +69,11 @@ func importCmd(c *cliconfig.ImportValues) error {
 		return errors.Errorf("too many arguments. Usage TARBALL [REFERENCE]")
 	}
 
-	if err := parse.ValidateFileName(source); err != nil {
-		return err
+	errFileName := parse.ValidateFileName(source)
+	errURL := parse.ValidURL(source)
+
+	if errFileName != nil && errURL != nil {
+		return errors.Errorf("%q, %q invalid tarball path", errFileName, errURL)
```

Comment:
We could use a multierror here, maybe?

Reply:
https://github.com/hashicorp/go-multierror

Reply:
:ok_hand: I'm looking